clients/venice-admin-tool/src/main/java/com/linkedin/venice/Arg.java

```diff
@@ -39,6 +39,7 @@ public enum Arg {
   STORAGE_NODE("storage-node", "n", true, "Helix instance ID for a storage node, eg. lva1-app1234_1690"),
   KEY("key", "k", true, "Plain-text key for identifying a record in a store"),
   OFFSET("offset", "of", true, "Kafka offset number"),
+  EXECUTION_ID("execution-id", "exec", true, "Execution ID of admin operation"),
```

Comment:
Can we rename `exec` to `eid` or something?

```suggestion
  EXECUTION_ID("execution-id", "eid", true, "Execution ID of admin operation"),
```